app/views/homes/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1><%= title.upcase %></h1>
```

Comment:
Edward.

I _love_ pedantry.

Fixed in 54f0441ed9c7ca0e30203e2d64ff2c335c1b487a